src/components/ToDoList/ToDoList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './todoList.css';
+import { ToDo } from '../ToDo';
+import { TodoType } from '../../types';
+
+export function ToDoList({ listOfTodos }) {
+  return (
+    <div className="todo-list">
+      <h2>To Do List:</h2>
+
+      {listOfTodos.map((user, index) => {
+        if (user.todo.length > 0) {
+          return (
+            <div
+              className="todo-list__todos"
+              key={user.id}
+            >
+              <p>
+                { 'User ' }
+                <span className="todo-list__username">
+                  {user.name}
+                  {' '}
+                </span>
+                needs to:
+              </p>
+              {listOfTodos[index].todo.map(item => (
+                <ToDo item={item} key={item.id} />
+              ))}
+            </div>
```

Comment:
now you double cycle, let's prepare a list of todos correctly in App, so you don't need 2 maps